source/common/res/features/toolkit-reports/main.js

```diff
@@ -40,15 +40,16 @@ const YNAB_NATIVE_CONTENT_SELECTOR = 'div.scroll-wrap';
         if ($('li.ynabtk-navlink-reports').length > 0) return;
 
         $('.nav-main > li:eq(1)').after(
-          $('<li>').append(
-            $('<li>', { class: 'ember-view ynabtk-navlink-reports' }).append(
+          // $('<li>').append(
+            // $('<li>', { class: 'ember-view ynabtk-navlink-reports' }).append(
```

Comment:
Would be good to remove the commented out code as it's not being used, but definitely not a blocker to merging.